pkg/apis/migration/v1alpha1/condition.go

```diff
@@ -16,6 +16,16 @@ const (
 	False = "False"
 )
 
+// Category
+// Required - Required for the `Ready` condition.
+// Error - Errors that block the `Ready` condition.
+// Warning - Warnings that block the `Ready` condition.
+const (
+	Error    = "Error"
```

Comment:
This introduces condition categories:
- `Error`
- `Required`
- `Warn`

@jortel if these are our only conditions, what category do we assign to good conditions? Do all good conditions fall under `Required`?

Reply:
Yes, I thought they would.